shoov/shoov.profile

```diff
@@ -77,6 +82,31 @@ function shoov_setup_og_permissions() {
 }
 
 /**
+ * Task callback; Setup content permissions.
+ *
+ * We do this here, late enough to make sure all content types were
+ * created.
+ */
+function shoov_setup_content_permissions() {
+  $content_type = 'repository';
+  $content_types = array_keys(node_type_get_types());
```

Comment:
I think there no need for all this. I believe we just need the `create repository content` permission. Anyway, Behat will confirm it :)